assembl/views/api2/discussion.py

```diff
@@ -511,37 +511,56 @@ def extract_taxonomy_csv(request):
     import assembl.models as m
     discussion = request.context._instance
     db = discussion.db
-    format = get_format(request)
     extracts = db.query(m.Extract).all()
     extract_list = []
-    fieldnames = ["Thematic", "Message", "Content Harvested", "Qualify by nature", "Qualify by action",
+    fieldnames = ["Thematic", "Message", "Content harvested", "Qualify by nature", "Qualify by action",
                   "Owner of the message", "Published on", "Harvester", "Harvested on", "Nugget"]
     for extract in extracts:
         thematic = db.query(m.Idea).filter(m.Idea.id == extract.idea_id).first()
```

Comment:
This query won't use the sqlalchemy identity map, so it will do the sql query for each extract, not very efficient.
please replace by `thematic = db.query(m.Idea).get(extract.idea_id)`